templates/master/01-master-kubelet/_base/files/kubelet.yaml

```diff
@@ -28,7 +28,6 @@ contents:
       APIPriorityAndFairness: true
       RotateKubeletServerCertificate: true
       DownwardAPIHugePages: true
-      OpenShiftPodSecurityAdmission: true
```

Comment:
Question: is this due to the api update?

Reply:
yes, it got moved in the openshift/api repo between 4.13 and 4.14. but now that i think about it, not sure why we had to move it in 4.13 here, iirc the tests were broken until we changed the template.